### PR TITLE
Brian/tooltip

### PR DESCRIPTION
**Pull request by briansmiley**
| :rocket: | This description was created by [Ellipsis](https://www.ellipsis.dev) for commit 45ed8fc8628683ea0210122dcf3a1faf5346fc61  | 
|--------|--------|

### Summary:
Introduced a new `Tooltip` component with customizable position and content, including example usage and Storybook stories, and removed outdated Storybook files.

**Key points**:
- Added `src/components/Tooltip.tsx` for a new `Tooltip` component.
- `Tooltip` supports `top`, `bottom`, `left`, and `right` positions.
- Tooltip visibility toggles on hover and focus events.
- Created `src/examples/TooltipExample.tsx` to demonstrate `Tooltip` usage.
- Added `src/stories/Tooltip.stories.ts` for Storybook integration with text and image tooltip examples.
- Removed old Storybook files: `src/stories/Header.stories.ts`, `src/stories/Header.tsx`, `src/stories/Page.stories.ts`, `src/stories/Page.tsx`.

----
Generated with :heart: by [ellipsis.dev](https://www.ellipsis.dev)

### CODE REVIEW

**Comment by briansmiley** [comment]
https://github.com/user-attachments/assets/2d75d5e4-631d-4e70-a308-93744b83cf3a

File: src/examples/TooltipExample.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import Tooltip from "../components/Tooltip";
```
Comment (jakezegil) [inline src/examples/TooltipExample.tsx:1]: Thanks for adding an example

File: src/components/Tooltip.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+}
+export default function Tooltip({
+  children,
+  tipContent,
+  position
+}: TooltipProps) {
+  //TODO: render the children as a subdiv with an on-hover that renders the tooltip in the indicated position
+  const [show, setShow] = useState(false);
+  //string dynamically generating the proper positioning class for the tooltip, e.g. "absolute ..." with positioning placing the tooltip subelement in the indicated position
+  const positionClass = useMemo(() => {
```
Comment (jakezegil) [inline src/components/Tooltip.tsx:16]: this definitely does not need a useMemo

File: src/components/Tooltip.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import React, { useMemo, useState } from "react";
+import "../index.css";
+interface TooltipProps {
+  children: React.ReactNode; //the page elements being wrapped which should display tooltip when hovered
+  tipContent: React.ReactNode; //the text/other content to render in the tooltip, passed in as a text/JSX/whatever prop
+  position: "top" | "bottom" | "left" | "right"; //the position of the tooltip relative to the wrapped element
```
Comment (jakezegil) [inline src/components/Tooltip.tsx:6]: nit: consider using an enum

**Review by jakezegil — APPROVED** [review]

File: src/components/Tooltip.tsx (hunk 3 above)
Comment (jakezegil) [reply src/components/Tooltip.tsx:6]:
```tsx
enum Postion {
 Top,
 Bottom,
 Left,
 Right
}

e.g.
const Position = {
 Top: 0,
 Bottom: 1,
 Left: 2,
 Right: 3
}
```

```
Position.Right === 3 // true
```

**Pull request merged**